README.md

```diff
@@ -1,6 +1,8 @@
 # Manifest for GloDroid
 
-Free and opensource AOSP based Android for the world's most accessible development platforms.  
+Free and opensource AOSP based Android for the world's most accessible development platforms.
+
+GloDroid [wiki](https://github.com/GloDroid/glodroid_manifest/wiki). You could find GloDroid community discord chat [here](https://discord.gg/2z5VUv6).
```

Comment:
Wiki page desn't have much useful information yet. Lets cleanup it first before publishing a link.